code/modules/projectiles/updated_projectiles/magazines/rifles.dm

```diff
@@ -82,11 +82,13 @@
 //-------------------------------------------------------
 //M41AE2 HEAVY PULSE RIFLE
 
-/obj/item/ammo_magazine/rifle/lmg
+/obj/item/ammo_magazine/lmg

```

Comment:
why

Reply:
Because I need the mags to not be able to be held by belts and such.
As far as I know, it was either this or having to specify every child of /obj/item/ammo_magazine/rifle/ for everything with an allowed list

Reply:
you should probably just use w_class although its a bit of mess at the moment with regular magazines being w_class 3

Reply:
Yeah just stick with the typepath change for now